config/jobs/kubernetes/sig-testing/kubernetes-kind-presubmits.yaml

```diff
@@ -319,7 +319,8 @@ presubmits:
 
   - name: pull-kubernetes-e2e-kind-ipv6
     optional: true
-    always_run: false # manual for testing new configurations
+    always_run: true
+    skip_report: true
```

Comment:
LGTM, but let's wait until we have more CI nodes that can run this. will discuss with the team this week.

Reply:
no rush, 
if this is part of the wg-infra effort and is an isolated cluster I can step up to help to maintain it

Reply:
> if this is part of the wg-infra effort

It isn't. (yet, and not soon)

> and is an isolated cluster 

It's not. it's in the prow.k8s.io build cluster, which is under google.

>  I can step up to help to maintain it

Appreciated, but:

There's not really anything to maintain, but having these nodes is not free, and every other job uses a large common non-snowflake pool of nodes for all kinds of CI.

There's a lot of prerequisites to sort out in wg-k8s-infra before we try to move prow 😞